src/goto-instrument/contracts/contracts.cpp

```diff
@@ -1553,3 +1582,107 @@ bool code_contractst::enforce_contracts(const std::set<std::string> &to_enforce)
   }
   return fail;
 }
+
+bool code_contractst::is_parameter_assign(
```

Comment:
Move all implementation to [contracts/utils.cpp](https://github.com/diffblue/cbmc/blob/develop/src/goto-instrument/contracts/utils.cpp)